### PR TITLE
Development setup documentation

### PR DESCRIPTION
**Pull request by jspaaks**
Added a document describing how to get your development setup up and running.

### CODE REVIEW

File: development-setup.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+Now, make the changes you want...
+
+```
+cd create-react-app/<somewhere>
+<make changes>
+```
+...and publish them to the ``sinopia`` registry as follows:
+
+```
+# TODO you may need to have added an npm user to the repo for this next step
```
Comment (sverhoeven) [inline development-setup.md]: Correct or change the configuration so $anonymous users can publish.
Please complete todo then PR is ready.

**Review by sverhoeven — APPROVED** [review]

**Comment by sverhoeven** [comment]
Please complete TODO of adduser so the whole sinopia based publishing flow is documented.

**Comment by sverhoeven** [comment]
Also add how to get back to https://registry.npmjs.org/

And how to no accidentally publish to an unwanted place.

**Comment by sverhoeven** [comment]
Add tl;dr section, copy from #1

**Comment by jspaaks** [comment]
Note "And how to no accidentally publish to an unwanted place." is still not really covered in this PR but see #29

**Comment by jspaaks** [comment]
refs #30

**Pull request merged**